Xero.NetStandard.OAuth2/Client/ApiException.cs

```diff
@@ -33,13 +33,13 @@ public class ApiException : Exception
         /// Gets or sets Retry-After value
         /// </summary>
         /// <value>The Retry-After (Http response header Retry-After).</value>
-        public int? RetryAfter { get; set; }
+        public int? RetryAfter { get; private set; }
```

Comment:
I am not sure if having a public set is part of convention or auto generation, but for correctness it should be private.